### PR TITLE
added question class

### PR DESCRIPTION
**Pull request by shlee2017**
corrected pr

### CODE REVIEW

File: front_end/Matching/HomeScreen.js [hunk 1]
```diff
@@ -1 +1,24 @@
-//Matching-Home
\ No newline at end of file
+//Matching-Home
+import React, {Component} from 'react';
+import {
+    SafeAreaView,
+} from 'react-native';
```
Comment (xumaple) [inline front_end/Matching/HomeScreen.js:5]: This can just be one line

Comment (xumaple) [inline front_end/Matching/HomeScreen.js:5]: This  can just be one line

File: front_end/Matching/HomeScreen.js [hunk 2]
```diff
@@ -1 +1,24 @@
...
+
+import QuestionList from './QuestionList';
+
+export default class HomeScreen extends Component {
```
Comment (xumaple) [inline front_end/Matching/HomeScreen.js:9]: I'm not sure what your structure looks like - but is this truly the app home screen? I suggest something like ```MatchingHomeScreen``` just to be more explicit

File: front_end/Matching/QuestionList.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            questions: props.questions,
+            //navigation: props.navigation for later
+        };
+    }
+
+    render(){
+        return (
+            <ScrollView>
+                {this.state.questions.forEach((q, i) => 
+                    <Question question={q} question_id = {i} key = {i} />
```
Comment (xumaple) [inline front_end/Matching/QuestionList.js:49]: keep spacing around equals sign consistent

File: front_end/Matching/MatchingHomeScreen.js [hunk 4]
```diff
@@ -1,12 +1,10 @@
 //Matching-Home
 import React, {Component} from 'react';
-import {
-    SafeAreaView,
-} from 'react-native';
+import { SafeAreaView, } from 'react-native';
```
Comment (xumaple) [inline front_end/Matching/MatchingHomeScreen.js]: bracket consistency lol

**Comment by hanssy130** [comment]
Rebasing.

**Comment by hanssy130** [comment]
Merge complete. Closing.

**Pull request merged**